perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/memberstabs/ChangeStatusTabItem.java

```diff
@@ -90,7 +90,6 @@ public Widget draw() {
 		final ListBox lb = new ListBox(false);
 		lb.addItem("VALID", "VALID");
 		lb.addItem("INVALID", "INVALID");
-		lb.addItem("SUSPENDED", "SUSPENDED");
```

Comment:
We can also remove whole usage of ``messageArea``, since no other status change takes a message. Consequently we can simplify ``json/membersManager/SetStatus.java`` and remove message param from that callback (constructor, variable etc.).

Reply:
removed